ui/packages/catalog-ui-search/src/main/webapp/react-component/container/loading-companion/loading-companion.tsx

```diff
@@ -50,10 +50,6 @@ export default class LoadingCompanionContainer extends React.Component<
     LoadingCompanionView.stopLoadingElement(this.ref.current)
   }
   render() {
-    return (
-      <Root innerRef={this.ref as React.RefObject<HTMLDivElement>}>
-        {this.props.children}
-      </Root>
-    )
+    return <Root ref={this.ref as any}>{this.props.children}</Root>
```

Comment:
Do we need a ticket made for retyping the refs after the styled typing issue is resolved?

Reply:
I think we'd have to help the styled type definitions out to actually get this fixed.